WePay/Models/WPConfig.m

```diff
@@ -55,7 +64,7 @@ - (NSDictionary *) toDict
     NSMutableDictionary *dict = [NSMutableDictionary dictionary];
     
     [dict setValue:self.clientId ? self.clientId : [NSNull null] forKey:@"clientId"];
-    [dict setValue:self.environment ? self.environment : [NSNull null] forKey:@"environment"];
+    [dict setValue:@(self.environment) forKey:@"environment"];
```

Comment:
The dictionary is only used for display purposes. The environment key does not need any special values, just whatever is set as the environment for the framework,